internal/services/containerapps/helpers/container_apps.go

```diff
@@ -1481,6 +1488,11 @@ func ContainerVolumeSchemaComputed() *pluginsdk.Schema {
 					Type:     pluginsdk.TypeString,
 					Computed: true,
 				},
+
+				"mount_options": {
+					Type:     pluginsdk.TypeString,
+					Required: false,
```

Comment:
Since this is the schema for the data source this needs to be `Computed` only
```suggestion
					Computed: true,
```

Reply:
This comment hasn't been addressed and is causing all the tests the fail

Reply:
Sorry, missed that one, should be fixed now.